src/test/resources/oracles/typescript/vscode-cliProcessMain-installExtensions.json

```diff
@@ -0,0 +1,51 @@
+{
+    "repositoryName": "vscode",
+    "filePath": "src/vs/code/node/cliProcessMain.ts",
+    "functionName": "installExtensions",
+    "functionStartLine": 128,
+    "startCommitName": "b0fb9f17da7b34ab122d9308c497b900fbe3cada",
+    "expectedResult": {
+        "613ea9ef4b3bf341755add37a4cd04904c3ea628": "Ybodychange",
+        "a50020021c0c528f0644e8ec814ef68e3ee8b9c6": "Ybodychange",
+        "e80300adbe5ae37f05c28ebda9ef05652d6d67fc": "Ybodychange",
+        "674807885b97ebe5342ea94e26833277a7339b5a": "Ymultichange(Yrename,Yreturntypechange,Ymodifierchange,Ybodychange)",
```

Comment:
I think this is a matter of opinion. You might even call this one here a method introduction.
The commit causing the confusion is [here](https://github.com/microsoft/vscode/commit/674807885b97ebe5342ea94e26833277a7339b5a)